test/can_spec.yml

```diff
@@ -0,0 +1,504 @@
+            atom:
+              ARG0:
+                slice: 0 + 48
+                type: uint64 little
+              ARG1:
+                slice: 48 + 16
+                type: uint16 little
+      map2:
+        version: 2.0B
+        baudrate: 1000000
+        extended: false
+        frame:
+          F:
+            key: 0x016
+            period: 101ms
+            atom:
+              ARG0:
+                slice: 0 + 10
+                type: int16 big
+              ARG1:
+                slice: 10 + 8
+                type: uint8 big
+              ARG2:
+                slice: 18 + 10
+                type: int16 big
+              ARG3:
+                slice: 28 + 8
+                type: uint8 big
+              ARG4:
+                slice: 36 + 10
+                type: int16 big
+              ARG5:
+                slice: 46 + 8
+                type: uint8 big
+              ARG6:
+                slice: 54 + 10
+                type: int16 big
+          G:
+            key: 0x017
+            period: 31ms
+            atom:
+              ARG0:
+                slice: 0 + 14
+                type: uint16 big
+              ARG1:
+                slice: 14 + 14
+                type: uint16 big
+              ARG2:
+                slice: 28 + 32
+                type: uint32 big
+          H:
+            key: 0x0A0
+            atom:
+              <<: &rms_temps
+                a_temp:
+                  slice: 0 + 16
+                  unit: decidegC
+                  type: int16 little
+                b_temp:
+                  slice: 16 + 16
+                  unit: decidegC
+                  type: int16 little
+                c_temp:
+                  slice: 32 + 16
+                  unit: decidegC
+                  type: int16 little
+                d_temp:
+                  slice: 48 + 16
+                  unit: decidegC
+                  type: int16 little
+          I:
+            key: 0xA2
+            atom:
+              ARG0:
+                slice: 0 + 16
+                type: int16 little
+              ARG1:
+                slice: 16 + 16
+                type: int16 little
+              ARG2:
+                slice: 32 + 16
+                type: int16 little
+              ARG3:
+                slice: 48 + 16
+                type: int16 little
+          J:
+            key: 0x0C2
+            slice: 0 + 8
+            frame:
+              AA:
+                key: 0
+                <<: &rms_bool_data
+                  atom:
+                    write_success:
+                      slice: 23 + 1
+                      type: bool little
+                    data:
+                      slice: 39 + 1
+                      type: bool little
+              BB:
+                key: 0x14
+                <<: *rms_bool_data
+                #Required to correctly parse the data with the script, as this parameter is constantly
+                #being transmitted over the CAN bus.
```

Comment:
you have lots of comments like this carried over from MY20
and the anchors here reference the RMS